infra/command/build-docker-image

```diff
@@ -26,10 +29,20 @@ do
       exit 1
       ;;
     --extension)
-      DOCKER_FILE_RPATH="${DOCKER_FILE_RPATH_BASE}.$2"
+      if [[ $2 == "1804" ]]; then
+        UBUNTU_CODENAME="bionic"
+      else
+        echo "Invalid extension"
+        exit 1
+      fi
       shift
       shift
       ;;
+    --codename)
+      UBUNTU_CODENAME=$2
+      DOCKER_TAG=$2
```

Comment:
I cannot understand this line. Are the rules for generating image names different when using `--extension` and `--codename`?

Reply:
`--extension` didn't set image names. `--codename` will set tag automatically if user don't set name explicitly by using `--tag` parameter or `DOCKER_IMAGE_NAME` environment variable

Reply:
I mean that the :`tag` part of image name, if I run this script with `--codename bionic`, the DOCKER_TAG will set as `bionic` here, so default image name will be `nnfw/nnas:bionic`, right?

Reply:
>so image name will be nnfw/nnas:bionic, right?

Right. 

I suggested to rename docker image (#4735), and it prepares renaming with ubuntu codename. When we support different architecture (ex. aarch64), tag name will include target architecture later.

Reply:
Aha, I understood. I have missed that issue. 👍